std/digest/murmurhash.d

```diff
@@ -500,28 +509,75 @@ struct MurmurHash3(uint size /* 32 or 128 */ , uint opt = size_t.sizeof == 8 ? 6
         // Buffer should never be full while entering this function.
         assert(bufferSize < Element.sizeof);
 
-        // Check if we have some leftover data in the buffer. Then fill the first block buffer.
+        // Check if we don't even fill up a single block buffer.
```

Comment:
This is very awkward wording. What about "check if we can fill up a single block buffer"?

Reply:
The true-branch with early return is for the opposite case, i.e., where the current bytes in the block buffer and the new bytes to be hashed don't add up to a full `Element` size.

Reply:
Not sure whether applicable here, but don't forget that for simple documentation matters patching someone else's PR is almost as easy as submitting a comment.

Reply:
I don't like the idea of editing someone's pull request unless it's necessary.

Reply:
@kinke "check if we can **'t** fill up a single block buffer?" ;-)

Reply:
Now: `Check if we don't fill up a full block buffer.`

Reply:
Just noticed `fill up a full ...` =>`Check if we don't fill up a whole block buffer.`